okhttp/src/jvmMain/kotlin/okhttp3/internal/http2/Http2Stream.kt

```diff
@@ -656,6 +657,8 @@ class Http2Stream internal constructor(
   }
 
   companion object {
+    internal fun windowThreshold(okHttpSettings: Settings) = okHttpSettings.initialWindowSize / 2
```

Comment:
Can remove, but it seems more than coincidental they match, so maybe should be a single definition.